include/deal.II/matrix_free/util.h

```diff
@@ -138,6 +149,22 @@ namespace internal
                   return {Quadrature<dim - 1>(),
                           QWitherdenVincentSimplex<dim - 1>(i)};
               }
+
+          for (unsigned int i = 1; i <= 3; ++i)
+            {
+              FE_SimplexP<dim> fe(i);
+              if (quad == Quadrature<dim>(fe.get_unit_support_points()))
+                {
+                  if (dim == 2)
+                    return {Quadrature<dim - 1>(
+                              fe.get_unit_face_support_points()), // line!
+                            Quadrature<dim - 1>()};
+                  else
+                    return {Quadrature<dim - 1>(),
+                            Quadrature<dim - 1>(
+                              fe.get_unit_face_support_points())};
+                }
```

Comment:
If you had a minute: As a follow-up, would you be willing to make `fe` `const`? That does not change anything other than make it easier for the reader to understand that all we do is *ask* the finite element something.

Reply:
Sorry, follow-up in #17230

Reply:
No need to apologize -- the follow-up is much appreciated!